static/app/components/replays/breadcrumbs/openReplayComparisonButton.tsx

```diff
@@ -29,6 +29,7 @@ export function OpenReplayComparisonButton({
     <Button
       role="button"
       size="xs"
+      analyticsEventKey="replay.details-hydration-modal-opened"
```

Comment:
Might want `analyticsEventName` as well, and i don't think you need to edit `replayAnalyticsEvents.tsx` if you're using these button props

https://develop.sentry.dev/analytics/#for-frontend-events

Reply:
right right forgot about that. should've reused those types somehow